README.md

```diff
@@ -567,3 +568,345 @@ notebook
+                                  Filter content results by Python version.
+  --title-contains TEXT           Filter content results by title.
+  --order-by [created|last_deployed]
+                                  Order content results.
+  -v, --verbose                   Print detailed messages.
+  --help                          Show this message and exit.
+
+$ rsconnect content search
+[
+  {
+    "max_conns_per_process": null,
+    "content_category": "",
+    "load_factor": null,
+    "cluster_name": "Local",
+    "description": "",
+    "bundle_id": "142",
+    "image_name": null,
+    "r_version": null,
+    "content_url": "https://connect.example.org:3939/content/4ffc819c-065c-420c-88eb-332db1133317/",
+    "connection_timeout": null,
+    "min_processes": null,
+    "last_deployed_time": "2021-12-02T18:09:11Z",
+    "name": "logs-api-python",
+    "title": "logs-api-python",
+    "created_time": "2021-07-19T19:17:32Z",
+    "read_timeout": null,
+    "guid": "4ffc819c-065c-420c-88eb-332db1133317",
+    "parameterized": false,
+    "run_as": null,
+    "py_version": "3.8.2",
+    "idle_timeout": null,
+    "app_role": "owner",
+    "access_type": "acl",
+    "app_mode": "python-api",
+    "init_timeout": null,
+    "id": "18",
+    "quarto_version": null,
+    "dashboard_url": "https://connect.example.org:3939/connect/#/apps/4ffc819c-065c-420c-88eb-332db1133317",
+    "run_as_current_user": false,
+    "owner_guid": "edf26318-0027-4d9d-bbbb-54703ebb1855",
+    "max_processes": null
+  },
+  ...
+]
+```
+
+See [this section](#searching-for-content) for more comprehensive usage examples
+of the available search flags.
+
+
+### Content Build
+
+> **Note:** The `rsconnect content build` subcommand requires RStudio Connect >= 2021.11.1
+
+RStudio Connect caches R and Python packages in the configured
+[`Server.DataDir`](https://docs.rstudio.com/connect/admin/appendix/configuration/#Server.DataDir).
+Sometimes, these package caches become stale and need to be rebuilt. This refresh
+occurs automatically when an RStudio Connect user visits the content via the
+RStudio Connect dashboard. Some content, however, is high priority or not accessed
+frequently via the dashboard (API content, emailed reports). In these cases, it is possible to
+preemptively build specific content items using the `rsconnect content build` subcommands.
+This way the user does not have to pay the build cost when the content is accessed next.
+
+The following are some common scenarios where performing a content build might be necessary:
+
+- OS upgrade
+- changes to gcc or libc libraries
+- changes to Python or R installations
+- switching from source to binary package repositories or vice versa
+
+> **Note:** The `content build` command is non-destructive, meaning that it does nothing to purge
+> existing packrat/python package caches before a build. If you have an
+> existing cache, it should be cleared prior to starting a content build.
```

Comment:
@aronatkins @akgold Added a note here to clarify for users that they are responsible for clearing the cache. Should we also point them to these docs or is that TMI? https://docs.rstudio.com/connect/1.5.14/admin/cli.html#migration-cli

Reply:
Linking feels like the right thing, and is the best we can do today.